tests/gateway/providers/test_mosaicml.py

```diff
@@ -91,58 +93,108 @@ def chat_response():
 
 
 @pytest.mark.parametrize(
-    "payload",
+    ("payload", "expected_llm_input"),
```

Comment:
```suggestion
    ("payload", "expected_input"),
```

nit: do we need `llm_`?

Reply:
It's certainly clearer: This is the thing we send to the LLM. `input` could mean the thing we send to the gateway.